### PR TITLE
Removed API docs for the latest version and deleted script that generated them

### PR DESCRIPTION
**Pull request by red2k18**
This PR does the following:

- [x] Removes API docs from the latest version
- [x] Removes the logic to generate API docs for the latest version

Before:

image

After:

image

API docs are not among the top 100 pages visited by the users in the last 90 days.

This change cuts down the build time by about 3 minutes.

### CODE REVIEW

File: scripts/generate_sphinx_docs.sh [hunk 1]
```diff
@@ -55,41 +55,9 @@ function build_python_source_in_docker() {
     popd
 }
 
-#######################################
```
Comment (blueandgold) [inline scripts/generate_sphinx_docs.sh]: I think that you need to delete this whole `.sh` file.  Also is this script referenced anywhere else?  Other scripts, documentations?  Do they need to be updated?

Comment (red2k18) [reply scripts/generate_sphinx_docs.sh]: Yes, the whole script can be removed. Deleted script and updated references. Time saved is 30 seconds (already added in the description).

**Comment by red2k18** [comment]
Yes, have tested by running the website locally.

File: README.md [hunk 2]
```diff
@@ -361,8 +361,6 @@ All of these different pieces are available and linked together properly within
 
 We use Docker throughout this build process to both install the Forseti Security libraries and process the code. This reduces the amount of dependencies necessary on the host machine to *only* Docker. There's no need for Python, Sphinx, or anything else to be modified on the host machine.
```
Comment (blueandgold) [inline README.md]: Please remove `Sphinx` in this sentence.

File: _data/doc_version_index.yml [hunk 3]
```diff
@@ -129,3134 +129,1257 @@ develop/index.md:
 - v2.24
 - v2.23
 develop/reference/.buildinfo:
```
Comment (blueandgold) [inline _data/doc_version_index.yml]: Should this line to `develop/reference` be removed as well?

File: README.md (hunk 2 above)
Comment (red2k18) [reply README.md]: Done

File: _data/doc_version_index.yml (hunk 3 above)
Comment (red2k18) [reply _data/doc_version_index.yml]: Added the auto generated file that removes all the references to develop/reference from `_data/doc_version_index.yml`. Have tested on my local; website renders perfectly fine.

**Pull request merged**